NuRadioReco/detector/RNO_G/RNO_season_2021.json

```diff
@@ -1300,7 +1300,7 @@
             "ant_orientation_theta": 0.0,
             "ant_position_x": -17.2,
             "ant_position_y": 29.097,
-            "ant_position_z": -93.72,
+            "ant_position_z": -92.72,
```

Comment:
This value does not seem correct, since the HPol is at ~ the same height on that string.

Reply:
good call. 

The calpulsers are indeed always deployed as the highest antenna of the helper strings.